tests/test_deployment.py

```diff
@@ -845,7 +845,7 @@ def test_failure(self, debugger):
 @pytest.mark.slow
 @pytest.mark.docker
 class TestDockerServer(object):
-    @pytest.mark.usefixtures("check_runbot")
+    @pytest.mark.skip(reason="Fails when run in the CI")
```

Comment:
If we don't get this figured out but the test works consistently locally, we could have pytest skip it based on the `CI` environment variable, I think.